.contributing/the-process.md

```diff
@@ -61,27 +61,24 @@ For each use-case there will be three types of digital goodness:
 
     -   Sequence Diagram
 
-Create: 
+Inside Google Drive Folder "x. *Section*: Sections Planning (CL)", create:
 
--   An Asana task for each use case, linked to the GitHub issue and Google doc (see below) 
+-   A Models file
 
--   A Drive folder for each use case, called "x. *Use Case* < *Section* (CL)" and in that folder, a Google Doc for each use case: 
+      -   Create a UML  - ([example](https://docs.google.com/drawings/d/1-X-aZdVrsuFItxlwkJqusOINdk0kYb9dl3fgLTQVz94/template/preview?usp=drive_web))
+      
+      -   Add title - "Models < x. *Use Case* < *Section* (CL)"
 
-    -   Models 
```

Comment:
creating the models file was still under the use case folder, when after discussion ws decided to be at the section level, so in the process doc, moved it up to so it wasnt a child of a use case, but a child of the section folder

Reply:
Should the model have a use case tied to it since it belongs to the section instead of a use case? Maybe we should just leave the use case off of the title? "Models <  *Section* (CL)"

Reply:
dadoi tam. good pickup @T-Davis! ill update it!!

Reply:
updated :)